skyvern/webeye/browser_factory.py

```diff
@@ -161,20 +161,26 @@ def update_chromium_browser_preferences(user_data_dir: str, download_dir: str) -
             f.write(preference_file_content)
 
     @staticmethod
-    def build_browser_args(proxy_location: ProxyLocation | None = None) -> dict[str, Any]:
+    def build_browser_args(proxy_location: ProxyLocation | None = None, **kwargs: dict) -> dict[str, Any]:
```

Comment:
@LawyZheng pls also take a look at this part

Reply:
Let's pass cdp_port explicitly